scylla/src/transport/session.rs

```diff
@@ -1650,7 +1650,7 @@ where
 
         let span = RequestSpan::new_batch();
 
-        let run_request_result = self
+        let run_query_result: RunRequestResult<NonErrorQueryResponse> = self
             .run_request(
                 statement_info,
                 &batch.config,
```

Comment:
🔧 Previous PR (which is included in this one change the name from using "query " to "request" but here you are changing it back. It looks like a mistake.

Reply:
Yeah, I wrote this code a while ago. The previous PR was created just today. I had to miss it during a rebase (or there had to be no conflict in this place, so I did not notice). Good catch